### PR TITLE
Added new typing animation component

### PR DESCRIPTION
**Pull request by fernandonevarez**
Added typing animation component to the home page.

### CODE REVIEW

File: src/components/homeHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Typewriter from "typewriter-effect";
+
+const HomeHeader = () => {
+  return (
+    <div>
+      CodeDevils - An ASU Student Organization{" "}
+      <Typewriter
+        options={{
+          strings: ["Devlope", "Create", "Network"],
```
Comment (reharri7) [inline src/components/homeHeader.tsx]: Change to `"Develop.", "Create.", "Network."`

**Review by reharri7 — CHANGES_REQUESTED** [review]
Please correct spelling

File: src/components/homeHeader.tsx (hunk 1 above)
Comment (fernandonevarez) [reply src/components/homeHeader.tsx]: alright 👍

**Review by fernandonevarez — COMMENTED** [review]
Made spelling changes.

**Review by reharri7 — APPROVED** [review]

**Comment by reharri7** [comment]
LGTM

**Pull request merged**